src/vivarium_nih_us_cvd/components/healthcare_visits.py

```diff
@@ -207,31 +313,29 @@ def visit_doctor(
             min_followup: minimum number of days out to schedule followup
             max_followup: maximum number of days out to schedule followup
         """
+
         # Update visit type
-        df[self.visit_type_column] = "none"
+        df[self.visit_type_column] = data_values.VISIT_TYPE.NONE  # Reset from last time step
         for k in visitors:
             df.loc[visitors[k], self.visit_type_column] = df[self.visit_type_column] + f";{k}"
 
-        # Update treatments
+        # Update treatments (all visit types go through the same treatment ramp)
         to_visit = pd.Index([])
         attended_visit_types = [
             visit_type
-            for visit_type in data_values.VISIT_TYPES
-            if visit_type != data_values.VISIT_TYPES.MISSED
+            for visit_type in data_values.VISIT_TYPE
+            if not visit_type in [data_values.VISIT_TYPE.MISSED, data_values.VISIT_TYPE.NONE]
```

Comment:
Why are you creating this data structure every time this function is called? Isn't it going to have the same values every time?

Reply:
Yes, good call, I'll move it out